types.go

```diff
@@ -79,6 +83,12 @@ type RWTable[Obj any] interface {
 	// write transaction return the fresh uncommitted modifications if any.
 	Table[Obj]
 
+	// RegisterInitializer adds an initializers to the table. Returns
+	// a function to mark the initializer done. Once all initializers are
+	// done, Table[*].Initialized() will return true.
+	// This should only be used before the application has started.
```

Comment:
What about adding an additional warning regarding the need to call the "done" function in a write txn?
Even if it is already stated for each function in `RWTable` interface, it might not be immediate for the returned function, since the user doesn't necessarily know the inner mechanics of the initializers registration.